### PR TITLE
Adding traits as used by IMPC

### PR DESCRIPTION
**Pull request by souzadevinicius**
This PR solves this [issue](https://github.com/obophenotype/bio-attribute-ontology/issues/176) partially.

### CODE REVIEW

**Review by matentzn — CHANGES_REQUESTED** [review]
Great job.

There are a number of irrelevant changes in this PR that belong in your VT alignment PR, like src/mappings/oba-vt.sssom.tsv

You should be careful when you create branches to _always create them off `main/master` branch_. You seem to be creating them of feature branches.

File: src/ontology/oba.Makefile [hunk 1]
```diff
@@ -54,7 +54,28 @@ modules/has_part.csv:  $(patsubst %, ../patterns/data/default/%.tsv, $(MODS))
 ## If you need to customize your Makefile, make
 ## changes here rather than in the main Makefile
 
-imports/hp_import.owl: $(TEMPLATEDIR)/external.owl
+#########################################
+### Generating all ROBOT templates ######
+#########################################
```
[the remark this answers is not in the document]
Comment (souzadevinicius) [reply src/ontology/oba.Makefile:59]: > I think you need to be more careful with your branches - these changes seem to belong to another branch!

Sorry

[the remark this answers is not in the document]
Comment (matentzn) [reply src/ontology/oba.Makefile:59]: Can you make a new PR, copying only the files you intended to change?

**Comment by matentzn** [comment]
I reverted the files.

File: src/ontology/oba.Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (matentzn) [reply src/ontology/oba.Makefile]: Done.

**Review by matentzn — CHANGES_REQUESTED** [review]
Please

- [x] Merge all tables in the PR into a single table (called impc.tsv) with an additional column "pattern name"
- [x] The additional column "pattern name" should have the name of the tsv files the row came from
- [x] The merge should be using use `defined_class` as primary key, but there are many other partially overlapping columns. Normal pandas merge?

**Comment by matentzn** [comment]
Do we still need this PR? Probably not right? We only care now about @ar-ibrahim curation of the IMPC terms!

**Comment by souzadevinicius** [comment]
> Do we still need this PR? Probably not right? We only care now about @ar-ibrahim curation of the IMPC terms!

No, we didn't

**Pull request closed without merging**